jekyll/Gemfile

```diff
@@ -1,14 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'asciidoctor', '>= 1.5.5'
-gem 'jekyll', '>= 3.6.2'
-gem 'jekyll-asciidoc', '>= 2.0.1'
-gem "algoliasearch-jekyll", '~> 0.9.0'
-gem 'pygments.rb'
-gem 'therubyracer'
-gem 'octopress-debugger'
-gem 'asciidoctor-github-include'
-
+gem 'asciidoctor', '1.5.6.1'
+gem 'jekyll', '>= 3.7.3'
+gem 'rouge'
 group :jekyll_plugins do
```

Comment:
do you need to change some build/whatever tools to make use of this `group`?

Reply:
No, it's pretty much decoration. The jekyll config file sees it and uses it, but they're also available as regular gems in the image.